protocols/market/contracts/Market.sol

```diff
@@ -197,23 +197,23 @@ contract Market is Ownable {
     * @param _startingPoint the staker to start at
     * @param _count the number of stakers to loop through
     */
-  function cleanExpiredIntents(address _startingPoint, uint256 _count) external {
+  function findExpiredIntents(address _startingPoint, uint256 _count) external view returns (address[] memory expiredIntents) {
```

Comment:
```    * @notice Loops through _count stakers from _startingPoint and removes any expired intents```
description is incorrect.

Reply:
missing return too.

Reply:
fixed all natspecs in [cce17f769590e48bcc40c49700b27ec6430da3e9](https://github.com/airswap/airswap-protocols/pull/78/commits/cce17f769590e48bcc40c49700b27ec6430da3e9) thanks!